backend/src/agent/persistence/main.go

```diff
@@ -43,6 +43,8 @@ var (
 	namespace                     string
 	ttlSecondsAfterWorkflowFinish int64
 	numWorker                     int
+	clientQPS                     float64
```

Comment:
Why do we use float64 here？ Looks like we use float32 for qps in other places？ Why don't we keep it consistent?

Reply:
NVM. There is no flag.Float32Var() method